notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>This tutorial demonstrates how to upload and deploy a PyTorch image model using a prebuilt serving container and how to make batch and online and batch predictions. </p><p> </p><p>Vertex AI provides prebuilt containers for serving predictions and explanations from trained model artifacts. Using a pre-built container is generally simpler than&nbsp;creating your own custom container for prediction.</p><p> </p><p>To learn more, see <a href="https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers" target="_blank">Pre-built containers for prediction</a>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>